src/pages/index.astro

```diff
@@ -1,127 +1,226 @@
-        fill="url(#paint0_linear_1805_24383)"></path>
-      <defs>
-        <linearGradient
-          id="paint0_linear_1805_24383"
-          x1="247.358"
-          y1="0"
-          x2="247.358"
-          y2="622.479"
-          gradientUnits="userSpaceOnUse"
-        >
-          <stop stop-opacity="0.9"></stop>
-          <stop offset="1" stop-opacity="0.2"></stop>
-        </linearGradient>
-      </defs>
-    </svg>
-    <h1>Welcome to <span class="text-gradient">Astro</span></h1>
-    <p class="instructions">
-      To get started, open the directory <code>src/pages</code> in your project.<br
-      />
-      <strong>Code Challenge:</strong> Tweak the "Welcome to Astro" message
-      above.
-    </p>
-    <ul role="list" class="link-card-grid">
-      <Card
-        href="https://docs.astro.build/"
-        title="Documentation"
-        body="Learn how Astro works and explore the official API docs."
-      />
-      <Card
-        href="https://astro.build/integrations/"
-        title="Integrations"
-        body="Supercharge your project with new frameworks and libraries."
-      />
-      <Card
-        href="https://astro.build/themes/"
-        title="Themes"
-        body="Explore a galaxy of community-built starter themes."
-      />
-      <Card
-        href="https://astro.build/chat/"
-        title="Community"
-        body="Come say hi to our amazing Discord community. ❤️"
-      />
-    </ul>
-  </main>
+<Layout>
+  <Navigation />
 </Layout>
 
 <style>
-  main {
+  body {
+    font-family: 'Open Sans', Arial, Helvetica, sans-serif;
+    background: #fff;
+    color: #000;
+    line-height: 1.5;
+    max-width: 1366px;
+    width: calc(100vw - 64px);
     margin: auto;
-    padding: 1rem;
-    width: 800px;
-    max-width: calc(100% - 2rem);
-    color: white;
-    font-size: 20px;
-    line-height: 1.6;
-  }
-  .astro-a {
-    position: absolute;
-    top: -32px;
-    left: 50%;
-    transform: translatex(-50%);
-    width: 220px;
-    height: auto;
-    z-index: -1;
   }
+
```

Comment:
duplication of _page.scss

Reply:
how you spot it? Show me how you do that!!!